app/views/condition/edit.erb

```diff
@@ -0,0 +1,61 @@
+<head>
+  <link rel="stylesheet" href = "//maxcdn.bootstrapcdn.com/bootstrap/3.3.1/css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="/condition/new.css">
+</head>
+
+<div class="transbox">
+  <form class="form-horizontal" action="/stations/<%= @trips.id %>" method="post">
+    <input type="hidden" name="_method" value="PUT" />
+    <fieldset>
+      <legend>Edit Weather Conditions Report</legend>
+      <div class="control-group">
+        <label class="control-label" name ="weather[date]" value="<%= @conditions.date %>"/>Date</label>
+        <div class="controls">
+          <input type="date" placeholder="date">
+        </div>
+      </div>
+      <div class="control-group">
+        <label class="control-label" name ="weather[max_temperature]" value="<%= @conditions.max_temperature %>"/>Temperature High</label>
+        <div class="controls">
+          <input type="integer" placeholder="high">
+        </div>
+      </div>
+      <div class="control-group">
+        <label class="control-label" name ="weather[min_temperature]" value="<%= @conditions.min_temperature %>"/>Temperature Low</label>
+        <div class="controls">
+          <input type="integer" placeholder="low">
+        </div>
+      </div>
+      <div class="control-group">
+        <label class="control-label" name ="weather[mean_humidity]" value="<%= @conditions.mean_humidity %>"/>Mean Humidity</label>
+        <div class="controls">
+          <input type="integer" placeholder="humidity">
+        </div>
+      </div>
+      <div class="control-group">
+        <label class="control-label" name ="weather[mean_visibility]" value="<%= @conditions.mean_visibility %>"/>Mean Visibility (miles)</label>
+        <div class="controls">
+          <input type="integer" placeholder="visibility (miles)">
+        </div>
+      </div>
+      <div class="control-group">
+        <label class="control-label" name ="weather[mean_wind_speed]" value="<%= @conditions.mean_wind_speed %>"/>Mean Wind Speed (mph)</label>
+        <div class="controls">
+          <input type="integer" placeholder="wind speed (mph)">
+        </div>
+      </div>
+      <div class="control-group">
+        <label class="control-label" name ="weather[precipitation]" value="<%= @conditions.precipitation %>"/>Precipitation (inches)</label>
+        <div class="controls">
+          <input type="integer" placeholder="precipitation (inches)">
```

Comment:
move this to input: name ="weather[precipitation]" value="<%= @conditions.precipitation %>"